confidence/models.py

```diff
@@ -13,15 +13,10 @@ class Missing(Enum):
     error = 'error'  #: raise an `AttributeError` for unconfigured keys
 
 
-class _NoDefault:
-    def __repr__(self) -> str:
-        return '(raise)'
-
-    __str__ = __repr__
-
-
-# overwrite _NoDefault as an instance of itself
-NoDefault = _NoDefault()
+_NoDefault = type("_NoDefault", (object,), {
```

Comment:
Without the additional assignment, this type can be named `NoDefault` now we don't need the underscore anymore.

Reply:
Done